packages/metascraper/test/unit/xss/xss.js

```diff
@@ -0,0 +1,28 @@
+'use strict'
+const fs = require('fs')
+const path = require('path')
+const metascraper = require(path.join(__dirname, '..', '..', '..'))([
+  require(path.join(__dirname, '..', '..', '..', '..', 'metascraper-title'))()
+])
+const should = require('should')
+
+describe('xss', () => {
+  // https://github.com/microlinkhq/metascraper/issues/96
+  // https://www.npmjs.com/advisories/603
```

Comment:
please remove the comment since the test is ensuring it will no more a warning 🙂